tests/acceptance/cli_test.py

```diff
@@ -510,7 +515,7 @@ def test_config_v3(self):
                     }, {
                         'target': '/anonymous', 'type': 'volume'
                     }, {
-                        'source': 'foobar',
+                        'source': 'foobar2',
```

Comment:
This can be reverted - as I pointed out above, it's valid for the same volume to be mounted in multiple places.